### PR TITLE
Refactor shutter action switch case for testing

### PR DESCRIPTION
**Pull request by evanmayer**
### Code coverage odyssey

Start breaking up big function `DoShutter()` by making one part of it it's own function. Test that function on its own, and replace the block in `DoShutter()` with the new function.

This is identical to what I have done already for `DoLock()` for the elevation lock state machine.

### CODE REVIEW

File: mcp/motors/actuators.c [hunk 1]
```diff
@@ -799,36 +799,23 @@ static int GetShutterData(struct ezbus* pBus, char who, int* lims, int* pos)
...
 
+
 /**
- * @brief Switch cases to call shutter move functions based on CommandData
+ * @brief Decides the shutter actuator action to take based on the goal.
+ * @param[in,out] shutter_state (uint32_t*) pointer to shutter_data.state, since it may
+ * need to be updated
+ * @param[in,out] shutter_goal (uint32_t*) pointer to CommandData.actbus.shutter_goal,
+ * since it may need to be updated
+ * @return action (uint32_t)
```
Comment (mxk62) [inline mcp/motors/actuators.c:809]: Doxygen will include function signature in the documentation it generates (see below)

Screenshot 2023-07-10 at 15-35-48 MCP motors_actuators c File Reference

As a result including them in the function description seem a little bit redundant. But it's just my opinion, not a fundamental objection. :wink:

**Comment by evanmayer** [comment]
I'm going to address these things quickly, so another commit incoming.

File: mcp/motors/actuators.c (hunk 1 above)
Comment (evanmayer) [reply mcp/motors/actuators.c:809]: Should I just remove `(uint32_t*)`, etc? The signature above has the names and types and stuff, but I like being able to note whether the param are inputs, outputs, or both.

Comment (mxk62) [reply mcp/motors/actuators.c:809]: I was just talking about `(uint32_t*)`. `[in, out]` is totally fine with me.

**Pull request merged**